config/scenario_config.csv

```diff
@@ -1,54 +1,54 @@
+SSP2EU-PkBudg1150;1,AMT;;;;;rcp26;9;globallyOptimal;1150;diffCurvPhaseIn2Lin;60;2100;9;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;on;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix3;2025;;SSP2EU-NDC;SSP2EU-Base;;SSP2EU-PkBudg1150: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 1150 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a weel below 2??C scenario, peak warming is allowed to be at or slightly above 2??C, at median climate sensitivity but returns to values below 2??C in at least 67?% of scenarios by the end of the century.
 # EU21 SSP2EU;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-SSP2EU-EU21-calibrate;0;calibrate;14;./config/21_regions_EU11/regionmapping_21_EU11.csv;./config/21_regions_EU11/extramapping_EU27.csv;;;;0;;;2100;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix1;2005;;;;;SSP2EU-EU21-calibration: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
-SSP2EU-EU21-NPi-calibrate;0;calibrate;14;./config/21_regions_EU11/regionmapping_21_EU11.csv;./config/21_regions_EU11/extramapping_EU27.csv;rcp45;;;0;NPi_Baseline;1;2100;9;;;;;;;;;;;NPi2018;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 3.266;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;heat;Mix1;2005;;;;;SSP2EU-EU21-NPi-calibrate: This reference policy/baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. 
+SSP2EU-EU21-calibrate;calibrate;calibrate;14;./config/21_regions_EU11/regionmapping_21_EU11.csv;./config/21_regions_EU11/extramapping_EU27.csv;;;;0;;;2100;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix1;2005;;;;;SSP2EU-EU21-calibration: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
+SSP2EU-EU21-NPi-calibrate;calibrate;calibrate;14;./config/21_regions_EU11/regionmapping_21_EU11.csv;./config/21_regions_EU11/extramapping_EU27.csv;rcp45;;;0;NPi_Baseline;1;2100;9;;;;;;;;;;;NPi2018;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 3.266;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;heat;Mix1;2005;;;;;SSP2EU-EU21-NPi-calibrate: This reference policy/baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
 SSP2EU-EU21-Base;1;;;./config/21_regions_EU11/regionmapping_21_EU11.csv;./config/21_regions_EU11/extramapping_EU27.csv;;;;0;;;2100;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix1;2005;;;;;SSP2EU-EU21-Base: This baseline calibration scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
```

Comment:
SSP2EU-EU21 scenarios should be part of the AMTs now

Reply:
And the "normal" SSP2EU scenarios (without EU21, so `SSP2EU-Base` et al)? Should they stay part of the AMTs?

Reply:
yes, SSP2EU-Base should be part of the AMTs, but SSP2EU-EU21-NPi-calibrate not part of the calibration (same reason as for SSP2EU)